### DISCUSSION TITLE
Do we have help `?` Under `NeogitLogView` buffer?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [NeogitOrg/neogit#595] [Q&A] [answered]
I found I can use shortcus like 

- `d` to `toggle diffview to view the changes made by that commit`
-  `<CR>` to `split right a window to view the changes`
-  `<Tab>` to `view more info about the commit under the cursor`

But is there shortcut to  
- `reset to the commit under the cursor`
- `rebase -i from HEAD till the commit under the cursor`
- ...

**Comment by CKolkey** [comment, marked as answer]
On the log/reflog view, we've implemented `revert` and `cherry pick` for the commit under the cursor, but not `rebase` or `reset`.

On the status buffer, `rebase`, `revert`, `cherry pick`, and `reset` should all use the commit under the cursor, if there is one.

**Reply by CKolkey** [reply]
But offering a help popup on other buffers is a good idea. I'd definitely add it to the TODO, or accept a PR :)

**Reply by LintaoAmons** [reply]
Thx~ Yeah, I hope I could be able to contribute something...But the code is a little bit difficult for me~
